Update hiring-process.md (#928)
publiccodenet/about: activities/recruitment/hiring-process.md

Between summer 2019 and summer 2021, most of our new hires discovered us through LinkedIn job ads.

## Processing reactions of people expressing interest

Questions about the content of the position are routed to chief executive. Questions about the hiring procedure and the conditions are routed to the operations coordinator who also takes care of processing and pre-selection.

## Planned recruitment and hiring process

1. Current job openings have no closing date and applications are considered on a rolling basis. When we find a suitable candidate, we hire and close the opening. This means applications are considered ‘by themselves’ and not ‘in competition’ with a certain set of other applications
2. OC confirms receipt to candidates and does first round of evaluation
3. OC brings possibly suitable candidates to the attention of CE and the role's future supervisor. These 2 make +/- judgement, without consulting each other. If judgement is not equal, a short discussion follows and judgement is made
4. OC notifies candidates (thank you or invitation for interview)

> 5. First interview: a lead codebase steward, or coordinator, depening on the role, and the OC, approximately 1 hr, partially on the basis of a pre-established list of desirable competences and other characteristics (i.e. fit for role, fit for organization (culture, growth, leadership, long/short term)). Half of the interview is spent presenting PublicCode.net, in the other half, the candidate has a chance to present themselves. Immediately after the interview the lead codebase steward and OC score the candidate on the items on the list and determine if the candidate should progress to a second interview. OC will contact the candidate to inform them

Reviewer: This is already not true for our communications editor.

```suggestion
5. First interview: a lead codebase steward or coordinator, depending on the role, and another senior staff person, approximately 1 hr, partially on the basis of a pre-established list of desirable competences and other characteristics (i.e. fit for role, fit for organization - including culture, growth, leadership, long/short term). Half of the interview is spent presenting the Foundation for Public Code, in the other half, the candidate has a chance to present themselves. Immediately after the interview the interviewers score the candidate on the items on the list and determine if the candidate should progress to a second interview. OC will contact the candidate to inform them
```

> 5. First interview: a lead codebase steward, or coordinator, depening on the role, and the OC, approximately 1 hr, partially on the basis of a pre-established list of desirable competences and other characteristics (i.e. fit for role, fit for organization (culture, growth, leadership, long/short term)). Half of the interview is spent presenting PublicCode.net, in the other half, the candidate has a chance to present themselves. Immediately after the interview the lead codebase steward and OC score the candidate on the items on the list and determine if the candidate should progress to a second interview. OC will contact the candidate to inform them

Reviewer: I've always disliked this formulation: it makes it sound like it'll be an hour of duelling powerpoints, rather than a dialogue driven by mutual curiosity.

> Half of the interview is spent presenting PublicCode.net, in the other half, the candidate has a chance to present themselves.

6. Second interview: CE with OC, also partially based on the pre-established list, and with attention to issues not discussed in the first interview

> 7. Approximately. 80-90% of the time is spent on questioning the candidate. Immediately after the interview OC and CE score the candidate, based on the list and other impressions gathered. OC takes this input to CE, and OC and CE determine if the candidate should receive an offer

Reviewer: The fact that the role's future supervisor isn't included in the decision seems pretty weird (and not scalable).

```suggestion
7. Approximately 80-90% of the time is spent on questioning the candidate. Immediately after the interview the interviewers score the candidate, based on the list and other impressions gathered. CE and OC decide if the candidate should be offered the job.
```

Author: I agree! This was very outdated and your fresh eyes on it greatly improved it :)

> 8. CE and OC determination of the offer, including salary range and benefits

Reviewer: ```suggestion
8. CE and OC determination of the specific offer, including salary range and benefits
```

9. Third interview, hopefully a formality, of the preferred candidate with OC and a BoD member. Decision on making an offer is made after third interview
10. OC makes the offer to the candidate and negotiates, always in consultation with CE and the director(s)

> 11. Signing of contract and starting by the new team member, supervised by their hierarchical supervisor

Reviewer: ```suggestion
11. Signing of contract and starting by the new team member, supervised by the role's supervisor
```
